### PR TITLE
First task

### PR DESCRIPTION
**Pull request by mend2000**
Added the Menu (Model, Control and View) and made some changes to Control and View classes.

### CODE REVIEW

**Comment by Edodums** [comment]
Great, the only thing you forgot was the Observer Pattern implementation

File: src/main/views/View.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        return primaryScreenBounds.getHeight() / BOUND_FACTOR;
+    }
+
+    void setScene() {
+        this.scene = new Scene(this.parent, getWidth(), getHeight());
+    }
+
+    void setParent(String filename) {
+        try {
+            this.parent = FXMLLoader.load(getClass().getResource(filename));
```
Comment (kieraCurtis) [inline src/main/views/View.java:45]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.

Comment (Edodums) [reply src/main/views/View.java:45]: That's true, thanks for the suggestion, good bot <3

Comment (kieraCurtis) [reply src/main/views/View.java:45]: Ok, I will recheck. Thanks a lot!

Comment (Edodums) [reply src/main/views/View.java:45]: Bot, check it in my next pull request

**Pull request merged**